### PR TITLE
added recipe for arctic 1.79.2

### PR DESCRIPTION
**Pull request by birnbaum**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by trendelkampschroer** [comment]
You can list me as a maintainer.

**Comment by mathiaseitz** [comment]
Happy to be a listed as a maintainer.

File: recipes/arctic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - decorator
+    - pandas
+    - pymongo >= 3.6.0
```
Comment (chrisburr) [inline recipes/arctic/meta.yaml]: I've already fixed it in master but this should be:
```suggestion
    - pymongo >=3.6.0
```
(If the formatting is wrong the feedstock creation fails)

Comment (birnbaum) [reply recipes/arctic/meta.yaml]: @xhochy @chrisburr Thanks for your help!

The repo https://github.com/conda-forge/arctic-feedstock is still empty though, is there anything I can do?

Comment (chrisburr) [reply recipes/arctic/meta.yaml]: Not really, I think [this job](https://travis-ci.org/conda-forge/staged-recipes/builds/538290357) should work (there was another version constraint issue and then a network glitch).

**Pull request merged**